spec/cli_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'spec_helper'
+require 'samwise'
+require 'json'
+require 'factory'
+require 'stringio'
+
```

Comment:
The spacing in this file should be fixed. Ruby programs should use 2 spaces (not tabs).